### DISCUSSION TITLE
Search Bar difficult to read in Dark Mode.

### DISCUSSION THREAD

**Discussion by HappyBullfrog** [bromite/bromite#1542] [General]
Bromite is the most capable and convenient Android browser I have ever used. However, I try to run all the software on my cell phone in Dark Mode to reduce eye strain. The Search Bar is difficult to read in Dark Mode. Is this the right place to request a fix? Thanks!

**Comment by uazo** [comment]
it depends. if you find the same thing in chromium it would be better to ask them directly, otherwise, you can try with a request here.

**Reply by HappyBullfrog** [reply]
The problem does NOT occur in Chrome nor in Vivaldi. In Bromite the Search Bar becomes all white when I try to type in it under Dark Mode, so I cannot see what I am typing. This happens both using Hacker's Keyboard or Gboard for entry. When not typing, the Search Bar seems fine; I can read it okay.
